config.tfvars

```diff
@@ -41,6 +41,18 @@ instance_disk_size = 50
 min_cluster_capacity = 1
 max_cluster_capacity = 5
 
+# If you desire to access the cluster with additional roles other than the one used for cluster creation,
+# you can define them below.
+#eks_additional_roles = [
+#  {
+#    rolearn  = "arn:aws:iam::ACCOUNT_ID:role/ROLE_NAME"
+#    username = "ROLE_NAME"
+#    groups = [
+#      "system:masters"
+#    ]
+#  }
+#]
+
```

Comment:
We can remove the commented block if is not needed anymore.

Reply:
Do you think it's worth adding our `dcd-private-sysadmin` role to [e2e test config template](https://github.com/atlassian-labs/data-center-terraform/blob/main/test/e2etest/test-config.tfvars.tmpl), so that we're able to access cluster if e2e test failed?

Reply:
> We can remove the commented block if is not needed anymore.

The block there is an example (it has the `ACCOUNT_ID` and `ROLE_NAME` as placeholders)

Reply:
> Do you think it's worth adding our `dcd-private-sysadmin` role to [e2e test config template](https://github.com/atlassian-labs/data-center-terraform/blob/main/test/e2etest/test-config.tfvars.tmpl), so that we're able to access cluster if e2e test failed?

That is a good point, I will create another PR for that.